src/redux/reservations/reservations.js

```diff
@@ -0,0 +1,51 @@
+export const FETCH_RESERVATIONS = 'BOOK-APPOINTMENT/RESERVATIONS/FETCH_RESERVATIONS';
+export const CREATE_RESERVATION = 'BOOK-APPOINTMENT/RESERVATIONS/CREATE_RESERVATION';
+export const DELETE_RESERVATION = 'BOOK-APPOINTMENT/RESERVATIONS/DELETE_RESERVATION';
+export const FETCH_SINGLE_RESERVATION = 'BOOK-APPOINTMENT/RESERVATIONS/FETCH_SINGLE_RESERVATION';
+export const UPDATE_MOTOR = 'BOOK-APPOINTMENT/RESERVATIONS/UPDATE_MOTOR';
+
+export const fetchReservations = (payload) => ({
+  type: FETCH_RESERVATIONS,
+  payload,
+});
```

Comment:
Great job so far. But there is no call to the API in this function. You can refactor the functions like the example below.

```
export const fetchReservations = (userId) => (dispatch) => {
  API.fetchReservations(userId, (response) => {
    dispatch({
       type: FETCH_RESERVATIONS,
       payload: response.data,
     });
  });
};
```
 Also, refactor all the functions below to dispatch a call to the API.

Do not forget to import the API from `redux/api.js`